src/Behat/Behat/Definition/Pattern/Pattern.php

```diff
@@ -17,59 +17,34 @@
  */
 final class Pattern
 {
-    /**
-     * @var string
-     */
-    private $canonicalText;
-    /**
-     * @var string
-     */
-    private $pattern;
-    /**
-     * @var int
-     */
-    private $placeholderCount;
+    public function __construct(
+        private string $suggestedMethodName,
+        private string $pattern,
+        private int $placeholderCount = 0,
+    ) {
+    }
```

Comment:
These previously had PHPDoc typehints, I would also be very surprised if anyone externally is creating a `Pattern` instance and certainly not with arguments that won't be able to cast to these types. I think therefore it's safe to make them strict.